<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schottky Spectroscopy Data Monitor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(35, 55, 59);
background-color: rgb(250, 250, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="GraphicsLayoutWidget" name="gTimeSeries"/>
      </item>
      <item>
       <widget class="PlotWidget" name="gSpectrum">
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="PlotWidget" name="gSpectrogram"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>scale</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="rLin">
             <property name="font">
              <font>
               <family>Roboto Condensed</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>linear</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rLog">
             <property name="font">
              <font>
               <family>Roboto Condensed</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>logarithmic</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>refresh</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QRadioButton" name="rAuto">
             <property name="font">
              <font>
               <family>Roboto Condensed</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>auto</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rManual">
             <property name="font">
              <font>
               <family>Roboto Condensed</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>manual</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>280</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableWidget" name="wParaTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Roboto Condensed</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="rowCount">
         <number>5</number>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <item row="0" column="0">
         <property name="text">
          <string>File Name</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>Timestamp in UTC</string>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>Reference Level</string>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>Sampling Rate</string>
         </property>
        </item>
        <item row="3" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="4" column="0">
         <property name="text">
          <string>IQ Pairs</string>
         </property>
        </item>
        <item row="4" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="vFileList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Roboto Condensed</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
